README.md

```diff
@@ -1,14 +1,16 @@
-# Tessel 2 Docs
-*Looking for Tessel 1 docs? Look [here](https://github.com/tessel/t1-docs).*
+Developer guide to Tessel 2 Docs Gitbook
+============
 
-Tessel is an open source project under open governance. Read more [here](https://github.com/tessel/project).
+# Tools
+Install the following command line tools using `npm i {{toolname}} -g`
 
-![](https://raw.githubusercontent.com/rwaldron/tessel-io/master/fritzing/tessel.png)
+* [gitbook](https://www.npmjs.com/package/gitbook)
+* [gitbook-summary](https://github.com/imfly/gitbook-summary)
 
-[Fritzing part for Tessel](https://github.com/rwaldron/tessel-io/tree/master/fritzing)
+# Running locally
 
-**Get involved:** The Tessel project welcomes contributors! Get started [here](https://github.com/tessel/project) and reach out if you have any questions.
+Clone, `npm install`, and `gitbook serve`
```

Comment:
There is no `package.json` in this repo, so `npm install` fails. There are two options for changing this:

- add a `package.json` with the `gitbook-cli` devDependency
- change this to say "Close, `npm install -g gitbook`, and `gitbook serve`

I think the less global tooling of contributors to install, the better.